wandb/apis/public.py

```diff
@@ -447,17 +447,16 @@ def create_run_queue(
         self,
         name: str,
         type: "RunQueueResourceType",
-        access: "RunQueueAccessType",
         entity: Optional[str] = None,
         prioritization_mode: Optional["RunQueuePrioritizationMode"] = None,
         config: Optional[dict] = None,
+        template_variables: Optional[dict] = None,
     ) -> "RunQueue":
         """Create a new run queue (launch).
 
         Arguments:
             name: (str) Name of the queue to create
             type: (str) Type of resource to be used for the queue. One of "local-container", "local-process", "kubernetes", "sagemaker", or "gcp-vertex".
-            access: (str) Access level for the queue. Either "project" or "user".
```

Comment:
I think this docstring is used to generate docs so I think this is a good place for us to document what this dictionary is supposed to look like.

Reply:
Another (better?) options is to allow for something like:
-t var1=x -t var2=y

but this gets tough because the types are harder to infer so I went this route. We can allow for a file, but then it should just be the `template_variables` key in the config file. Happy to drop this option

Reply:
Fixed